pkg/model/manifest.go

```diff
@@ -948,13 +948,6 @@ func (manifest *Manifest) ExpandEnvVars() error {
 				return err
 			}
 		}
-		for idx, cmd := range manifest.Destroy.Commands {
-			cmd.Command, err = envsubst.String(cmd.Command)
-			if err != nil {
-				return errors.New("could not parse env vars")
-			}
-			manifest.Destroy.Commands[idx] = cmd
```

Comment:
@jLopezbarb I am curious if there was a reason to have this code in place

Reply:
@pchico83 @jLopezbarb This PR was the first one where we included the env var expansasion for deploy and destroy commands: https://github.com/okteto/okteto/pull/2234

Then, in this PR we changed the approach to not expand the env vars in deploy commands, but we left the destroy commands: https://github.com/okteto/okteto/pull/2653

I think it was the first attempt of having variable expansion for build env vars instead of injecting them in the environment when executing the commands. At some point, we had some error in deploy commands and we removed it, but we didn't do the same for destroy commands

Reply:
Thanks @ifbyol for this brilliant analysis :-)
That gives us even more confidence that this change is totally safe.